index.html

```diff
@@ -0,0 +1,180 @@
+          </div>
+          <div class="logo-description">
+            <p>Rails is a framework that provides default structures for databases.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/devops-2 (1).svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Devops</h4>
+          </div>
+          <div class="logo-description">
+            <p>DevOps improves and shortens the systems development life cycle.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/kotlin-1.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Kotlin</h4>
+          </div>
+          <div class="logo-description">
+            <p>Kotlin is a cross-platform high-level programming language with type inference.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/vue-9.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Vue.js</h4>
+          </div>
+          <div class="logo-description">
+            <p>Vue.js is a JavaScript framework for building user interfaces and single-page applications.</p>
+          </div>
+        </div>
+      </div>
+      <a href="#">Join Our Program</a>
+    </section>
+
+    <section class="speakers">
+      <h2>Featured Speakers</h2>
+      <div id="speaker-cards"></div>
+    </section>
+
+    <section class="partners">
+      <h2>Partners</h2>
+      <div class="partner-logos">
+        <img src="./images/apple-13.svg" alt="">
+        <img src="./images/google-1-1.svg" alt="">
+        <img src="./images/meta-1.svg" alt="">
+        <img src="./images/microsoft.svg" alt="">
+        <img src="./images/oracle-6.svg" alt="">
+      </div>
+    </section>
+
+    <footer>
+      <div class="main-footer">
+        <h2>DEVSPARK</h2>
+        <div class="company-contacts">
+          <h4>Get In Touch</h4>
+          <div class="contact">
+            <p>Nairobi, Kenya <span><i class="fa-sharp fa-solid fa-location-dot"></i></span></p>
+            <p><a href="mailto:newtonombese1@gmail.com">newtonombese1@gmail.com</a> <span><i class="fa-solid fa-envelope"></i></span></p>
+            <p><a href="https://wa.me/254758982500">+254758982500</a> <span><i class="fa-solid fa-phone"></i></span></p>
+          </div>
+        </div>
+      </div>
+      <div class="company-socials">
+        <a href="">
+          <i class="fa-brands fa-facebook socials"></i>
+        </a>
+        <a href="">
+          <i class="fa-brands fa-instagram socials"></i>
+        </a>
+        <a href="">
+          <i class="fa-brands fa-twitter socials"></i>
+        </a>
+        <a href="">
+          <i class="fa-brands fa-youtube socials"></i>
+        </a>
+      </div>
+    </footer>
```

Comment:
- [ ] Please make sure that your footer design is similar to the original design for both the home and about pages. There are a few differences. Please note that it is very important to stick to the original design as much as possible :smile: 

#### Your design
![footer](https://github.com/NewtonOmbese/Newton-Capstone-Project/assets/6615394/bb2fad3e-c8bb-4bd4-8098-25e070c9c6f6)


#### The original design
![Screenshot from 2022-10-28 18-41-53](https://user-images.githubusercontent.com/6615394/198709533-f9ce48e0-df96-4c8a-b29b-ffea117dcfcf.jpg)